docs/examples/code-editor/codeEditor.js

```diff
@@ -0,0 +1,55 @@
+import { $, loadingSpinner, backdrop, iframe } from './common.js'; // Import common functions
+
+// Get the code editor elements
+document.addEventListener('DOMContentLoaded', () => {
+    const htmlCode = document.getElementById('htmlCode');
```

Comment:
Instead of `getElementById` maybe use `$` instead? Also, could have an array with the IDs then have the call to `$` inside the `forEach`


Alternately should it take the exported vars from common.js?